src/selection.js

```diff
@@ -149,31 +161,37 @@ export default class Selection extends Cursor {
 
   makeBold () {
```

Comment:
I think we should extend the toggleCustom as well

Reply:
We can do this. How would you configure it?
From the top of my head it could look like this:
```
customElements: [{
  handle: 'foo',
  ...,
  trim: true | false
}]
```

Since for custom elements this is a project config entry though the change would also be in the server, editor and framework. Is this what you imagine here?

Reply:
I see we don't really have any config in the editablejs for the custom element. So either we would add it with a tag and attributes as an indicator or we would add an option to the toggleCustom function to trim it. I think that would be the best way then we could add the config to the project config on our side.

I also just saw that we can't overwrite the trim config out of our editor.